src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointEvidence.java

```diff
@@ -240,31 +315,46 @@ public MateUnmapped read( final Kryo kryo, final Input input, final Class<MateUn
 
     @DefaultSerializer(InterContigPair.Serializer.class)
     public static final class InterContigPair extends BreakpointEvidence {
-        private final int mateContigIndex;
-        private final int mateStartPosition;
+        private final boolean mateReverseStrand;
+        private final SVInterval target;
 
         InterContigPair( final GATKRead read, final ReadMetadata metadata ) {
             super(read, metadata);
-            this.mateContigIndex = metadata.getContigID(read.getMateContig());
-            this.mateStartPosition = read.getMateStart();
+            final int mateContigIndex = metadata.getContigID(read.getMateContig());
+            final int mateStartPosition = read.getMateStart();
+            mateReverseStrand = read.mateIsReverseStrand();
+            final int medianFragmentSize = metadata.getStatistics(read.getReadGroup()).getMedianFragmentSize();
+            target = new SVInterval(mateContigIndex,
```

Comment:
We might document that due to the absence of the mate's cigar, we're doing some fudging.  Also, that we're assuming that the mate's length == the read's length.